draft-lmbdhk-quic-multipath.md

```diff
@@ -606,32 +596,17 @@ Path Identifier: An identifier of the path, which is formatted as shown in {{fig
 ~~~
 {: #fig-path-identifier-format title="Path Identifier Format"}
 
-Note: If the receiver of the PATH_STATUS frame is using non-zero length Connection ID on that path,
-endpoint SHOULD use type 0x00 for path identifier in the control frame. If the receiver of the PATH_STATUS frame
+Note: If the receiver of the PATH_ABANDON frame is using non-zero length Connection ID on that path,
+endpoint SHOULD use type 0x00 for path identifier in the control frame. If the receiver of the PATH_ABANDON frame
 is using 0-length Connection ID, but the peer is using non-zero length Connection ID on that path, endpoints
 SHOULD use type 0x01 for path identifier. If both endpoints are using 0-length Connection IDs on that path,
 endpoints SHOULD only use type 0x02 for path identifier.
 
-Path Status sequence number: A variable-length integer specifying the sequence number assigned for this PATH_STATUS frame.
-There is a different path status sequence number space for each path.
-
-Available values of Path Status field are:
-
-- 0: Abandon
-- 1: Standby
-- 2: Available
-
-If the value of Path Status field is 2 ("available"), the receiver side can use
-the Path Priority field to express the priority weight of a path for the peer.
-
-Frames may be received out of order. A peer MUST ignore an incoming PATH_STATUS
-frame if it previously received another PATH_STATUS frame for the same Path
-Identifier with a sequence number equal to or higher than the sequence number
-of the incoming frame.
+Error code: TBD
+Reason Phrase: TBD
 
```

Comment:
How about just adapting the text from QUIC Connection_Close frame:
```
Error Code:
: A variable-length integer that indicates the reason for closing this connection.

Reason Phrase Length:
: A variable-length integer specifying the length of the reason phrase in bytes.
  Because an ABANDON_FRAME frame cannot be split between packets, any limits
  on packet size will also limit the space available for a reason phrase.

Reason Phrase:
: Additional diagnostic information for the closure. This can be zero length if
  the sender chooses not to give details beyond the Error Code value.
  This SHOULD be a UTF-8 encoded string {{!RFC3629}}, though the frame
  does not carry information, such as language tags, that would aid comprehension
  by any entity other than the one that created the text.
```

The error code will often be "No Error". However, some issues can happen, such as closing a path because resource are exceeded, or because there are too many transmission errors, or anything else. Having error codes and an optional reason phrase will help diagnostic and debugging.